diff --git a/config.h b/config.h
index 946a40d..1e3f952 100644
--- a/config.h
+++ b/config.h
@@ -185,7 +185,7 @@ static const Key keys[] = {
     { MODKEY,               XK_a,       togglegaps,     {0} },
     { MODKEY|ShiftMask,     XK_a,       defaultgaps,    {0} },
     { MODKEY,               XK_s,       togglesticky,   {0} },
-    { MODKEY,               XK_d,       spawn,          SHCMD("dmenu_run -p 'Run: '") },
+    { MODKEY,               XK_d,       spawn,          SHCMD("dmenu_run") },
     /* { MODKEY,               XK_e,       spawn,      SHCMD("$HOME/.local/bin/scripts/emacs/emacs-daemon.sh start") }, */
     /* { MODKEY|ShiftMask,     XK_e,       spawn,      SHCMD("$HOME/.local/bin/scripts/emacs/emacs-daemon.sh kill") }, */
     { MODKEY,               XK_n,       spawn,      SHCMD("emacsclient -c --frame-parameters '((title . \"Emacs\"))'") },
@@ -196,11 +196,11 @@ static const Key keys[] = {
     { MODKEY|ShiftMask,     XK_m,       spawn,      SHCMD("$HOME/.local/bin/scripts/theme") },
     { MODKEY|ShiftMask,     XK_x,       spawn,      SHCMD("$HOME/.local/bin/scripts/sysact") },
 
-    { MODKEY|ShiftMask,     XK_p,                spawn,      SHCMD("playerctl -F play-pause") },
-    { MODKEY|ShiftMask,     XK_bracketleft,      spawn,      SHCMD("playerctl -F previous") },
-    { MODKEY|ShiftMask,     XK_bracketright,     spawn,      SHCMD("playerctl -F next") },
+    { MODKEY|ShiftMask,     XK_p,                spawn,      SHCMD("playerctl play-pause") },
+    { MODKEY|ShiftMask,     XK_bracketleft,      spawn,      SHCMD("playerctl previous") },
+    { MODKEY|ShiftMask,     XK_bracketright,     spawn,      SHCMD("playerctl next") },
 
-    { MODKEY|ShiftMask,     XK_b,       spawn,      SHCMD("boomer") },
+    { MODKEY|ShiftMask,     XK_b,       spawn,      SHCMD("coomer") },
     { MODKEY,               XK_f,       togglefullscr,  {0} },
     { MODKEY,               XK_h,       setmfact,   {.f = -0.05} },
     /* J and K are automatically bound above in STACKEYS */
@@ -223,18 +223,18 @@ static const Key keys[] = {
     { 0,                XK_Print,   spawn,      SHCMD("maim $HOME/Pictures/screenshots/pic-full-$(date '+%y%m%d-%H%M-%S').png") },
     { ShiftMask,        XK_Print,   spawn,      SHCMD("$HOME/.local/bin/scripts/maimpick") },
 
-    { 0, XF86XK_AudioMute,          spawn,      SHCMD("pactl set-sink-mute @DEFAULT_SINK@ toggle; kill -44 $(pidof dwmblocks)") },
-    { 0, XF86XK_AudioRaiseVolume,   spawn,      SHCMD("pactl set-sink-volume @DEFAULT_SINK@ +10%; kill -44 $(pidof dwmblocks)") },
-    { 0, XF86XK_AudioLowerVolume,   spawn,      SHCMD("pactl set-sink-volume @DEFAULT_SINK@ -10%; kill -44 $(pidof dwmblocks)") },
-    { 0, XF86XK_AudioMicMute,       spawn,      SHCMD("pactl set-source-mute @DEFAULT_SOURCE@ toggle") },
+    { 0, XF86XK_AudioMute,          spawn,      SHCMD("wpctl set-mute @DEFAULT_AUDIO_SINK@ toggle; kill -44 $(pidof dwmblocks)") },
+    { 0, XF86XK_AudioRaiseVolume,   spawn,      SHCMD("wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%+; kill -44 $(pidof dwmblocks)") },
+    { 0, XF86XK_AudioLowerVolume,   spawn,      SHCMD("wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%-; kill -44 $(pidof dwmblocks)") },
+    { 0, XF86XK_AudioMicMute,       spawn,      SHCMD("wpctl set-mute @DEFAULT_AUDIO_SOURCE@ toggle") },
 
     { 0, XF86XK_AudioPrev,      spawn,      {.v = (const char*[]){ "playerctl", "previous", NULL } } },
     { 0, XF86XK_AudioNext,      spawn,      {.v = (const char*[]){ "playerctl", "next", NULL } } },
     { 0, XF86XK_AudioPause,     spawn,      {.v = (const char*[]){ "playerctl", "pause", NULL } } },
     { 0, XF86XK_AudioPlay,      spawn,      {.v = (const char*[]){ "playerctl", "play", NULL } } },
 
-    { 0, XF86XK_MonBrightnessUp,    spawn,      {.v = (const char*[]){ "xbacklight", "-inc", "5", NULL } } },
-    { 0, XF86XK_MonBrightnessDown,  spawn,      {.v = (const char*[]){ "xbacklight", "-dec", "5", NULL } } },
+    { 0, XF86XK_MonBrightnessUp,    spawn,      {.v = (const char*[]){ "brightnessctl", "s", "5%+", NULL } } },
+    { 0, XF86XK_MonBrightnessDown,  spawn,      {.v = (const char*[]){ "brightnessctl", "s", "5%-", NULL } } },
 };
 
 /* button definitions */
